### PR TITLE
Link OpenCL Tooling blog into the resource page.

### PR DESCRIPTION
**Pull request by AnastasiaStulova**

### CODE REVIEW

File: api/opencl/resources.md [hunk 1]
```diff
@@ -23,12 +23,16 @@ Khronos also hosts an [OpenCL Community Resource Page](https://www.khronos.org/o
...
 *   [Libclc](https://github.com/llvm/llvm-project/tree/master/libclc) is a generic and portable implementation of OpenCL builtin function libraries for OpenCL 1.1 - and some functions from later versions of OpenCL can be found there too
 *   The [SPIRV-LLVM Translator](https://github.com/KhronosGroup/SPIRV-LLVM-Translator), is a library and tool for translating between LLVM IR and SPIR-V
 *   The open source [clspv compiler](https://github.com/google/clspv) and [clvk runtime layer](https://github.com/kpet/clvk) enable OpenCL applications to be executed on Vulkan
 *   [SPIR-V Tools](https://github.com/KhronosGroup/SPIRV-Tools) provide a set of utilities to process SPIR-V modules including an optimizer, linker, (dis-)assembler, and validator
 
+The following [blog](https://www.khronos.org/blog/offline-compilation-of-opencl-kernels-into-spir-v-using-open-source-tooling) contains detailed description of how to compile OpenCL kernels offline into SPIR-V using the available open source tools.
```
Comment (bashbaug) [inline api/opencl/resources.md:34]: Very minor wordsmithing nit: there is no "following" blog.  Maybe just change to:

"This [blog]..."?

**Review by bashbaug — COMMENTED** [review]
One minor wording nit, otherwise LGTM.

File: api/opencl/resources.md (hunk 1 above)
Comment (AnastasiaStulova) [reply api/opencl/resources.md]: Good point. I have updated it now!

**Comment by outofcontrol** [comment]
one more minor grammatical fix:
The following [blog](https://www.khronos.org/blog/offline-compilation-of-opencl-kernels-into-spir-v-using-open-source-tooling) contains a detailed description of how...

or 

The following [blog](https://www.khronos.org/blog/offline-compilation-of-opencl-kernels-into-spir-v-using-open-source-tooling) contains detailed descriptions of how... 

Former for one description, latter for 2+ descriptions.

**Comment by AnastasiaStulova** [comment]
> one more minor grammatical fix:
> The following [blog](https://www.khronos.org/blog/offline-compilation-of-opencl-kernels-into-spir-v-using-open-source-tooling) contains a detailed description of how...
> 
> or
> 
> The following [blog](https://www.khronos.org/blog/offline-compilation-of-opencl-kernels-into-spir-v-using-open-source-tooling) contains detailed descriptions of how...
> 
> Former for one description, latter for 2+ descriptions.

I think the first one makes more sense. I have updated it now! Thanks!

**Review by bashbaug — APPROVED** [review]
LGTM, thanks!

**Pull request merged**